opsi/modules/draw/__init__.py

```diff
@@ -7,7 +7,7 @@
 from opsi.manager.types import Contours, Mat, MatBW
 
 from .fps import DrawFPS
-from .shapes import DrawCircles
+from .shapes import *
```

Comment:
Generally, you should call specifically instead of using `*` when there isn't a large amount of things to import because it is faster.

Reply:
Ok, I will fix that. I also have discovered that I misunderstood the parameters for circle detection, so I renamed them to be more accurate.